ash-window/Cargo.toml

```diff
@@ -14,15 +14,15 @@ exclude = [".github/*"]
 workspace = ".."
 
 [dependencies]
-ash = { path = "../ash", version = "0.35", default-features = false }
+ash = { path = "../ash", version = ">=0.34, <=0.36", default-features = false }
 raw-window-handle = "0.3.4"
 
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
 raw-window-metal = "0.1"
 
 [dev-dependencies]
 winit = "0.19.4"
-ash = { path = "../ash", version = "0.35", default-features = false, features = ["linked"] }
+ash = { path = "../ash", version = ">=0.35, <=0.36", default-features = false, features = ["linked"] }
```

Comment:
Note though that we renamed `Entry::new()` to `Entry::linked()` in `0.35`, as such this example isn't compatible with `0.34`. Perhaps we can also just keep a fixed `0.36` in here though...

Reply:
I think a fixed version for the examples makes sense from a maintenance perspective.

Reply:
I presume that - and the approval below - is a yay for trying this on `ash-window`'s `ash` dependency though?

Reply:
Yeah, I'm all in favor of reduced toil. We'll have to be careful not to introduce new idioms, especially without CI coverage, but I think the risk is low considering the lib's narrow scope.

Reply:
Cool, let's see how this works out over time. If mistakes happen we can always re-evaluate and most likely come up with a simple CI test that ensures compatibility.